api/api.go

```diff
@@ -348,7 +348,7 @@ func (h *caHandler) ProvisionerKey(w http.ResponseWriter, r *http.Request) {
 func (h *caHandler) Roots(w http.ResponseWriter, r *http.Request) {
 	roots, err := h.Authority.GetRoots()
 	if err != nil {
-		WriteError(w, errs.ForbiddenErr(err))
+		WriteError(w, errs.ForbiddenErr(err, "error getting roots"))
```

Comment:
If this is the error returned to the user, that doesn't seem useful / actionable.

Reply:
This method never fails, but if it does and is an `errs.Error` that is the one that will be returned.

We don't have a lot of context of want went wrong there. So I'm messaging the only thing we know.